internal/runnerinstall/k8s.go

```diff
@@ -110,6 +112,24 @@ func (i *K8sRunnerInstaller) Install(ctx context.Context, opts *InstallOpts) err
 		return err
 	}
 
+	// Update the Hashicorp Helm repo before we attempt to install it
+	chartRepo, err := repo.NewChartRepository(&repo.Entry{
+		Name: "hashicorp",
+		URL:  "https://helm.releases.hashicorp.com",
```

Comment:
Not sure if we reference these values any where but does it make sense to make this a Const string some where in the case where this changes? Then it's changed in 1 place instead of many.